cmd/bbolt/command_page_item.go

```diff
@@ -0,0 +1,148 @@
+package main
+
+import (
+	"errors"
+	"fmt"
+	"io"
+	"os"
+	"strconv"
+
+	"github.com/spf13/cobra"
+	"github.com/spf13/pflag"
+
+	"go.etcd.io/bbolt/internal/common"
+	"go.etcd.io/bbolt/internal/guts_cli"
+)
+
+const (
+	pageItemUsage = `usage: bolt page-item [options] PATH pageid itemid
```

Comment:
would you kindly build your branch locally

```
make build
```

then paste here the output of 

```
./bin/bbolt page-item -h
```

I am curios as the long msg has duplicates with the short msg 

apologies for any inconveniences